Felienne/hedy
# [BUG FIX] Quiz answer 4 no longer always selected

**Description**
We fix a weird bug where the answer container of answer 4 would always be selected when switching a quiz question. The cause is still unknown, but by removing the `active` attribute in the answer loading loop instead of globally the issue seems to be fixed.

**Fixes**
This PR fixes #2860.

**How to test**
Navigate to quiz level 1 question 1 and answer, verify that when going to the next question answer 4 is no longer automatically selected.
